src/main/io/gps.c

```diff
@@ -2670,6 +2669,11 @@ float getGpsDataFrequencyHz(void)
     return gpsDataFrequencyHz;
 }
 
+float getGpsCosLat(void)
+{
+    return GPS_cosLat;
```

Comment:
Should GPS_cosLat be set to static to forbid access to it from other files? So, the getter would get sense

Reply:
Thanks.  Yes, I think so.
I didn't look closely at `GPS_cosLat` but your comment is, I think, correct.  It's value is only calculated once, on arming, if a home point can be derived, otherwise it will be 1.0. In an extremely long GPS Rescue the scaling factor won't change, as it should, but typically Betaflight won't be doing such a long rescue that it would matter.
Also I don't really understand the old comments :-)